### ISSUE TITLE
Update uap-core to get Microsoft Edge recognized

### ISSUE THREAD

**Issue by mzgol** [3rd-Eden/useragent#76]
Microsoft Edge is now recognized correctly by uap-core after https://github.com/ua-parser/uap-core/pull/59. All is needed here (I guess?) is to run the update script. :)

Currently:
```js
var useragent = require('useragent');
useragent.parse('Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.9600');
/*
{ family: 'IE',
  major: '12',
  minor: '9600',
  patch: '0',
  source: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.9600' }
*/
useragent(true);
useragent.parse('Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.9600');
/*
{ family: 'Edge',
  major: '12',
  minor: '9600',
  patch: '0',
  source: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.9600' }
*/
```

**Comment by nicolasmccurdy** [comment]
:+1:

**Comment by 3rd-Eden** [comment]
Released 2.1.7

**Comment by mzgol** [comment]
Thanks!
